AndroidAnnotations/androidannotations-core/androidannotations/src/test/java/org/androidannotations/databinding/R.java

```diff
@@ -0,0 +1,9 @@
+package org.androidannotations.databinding;
+
+public class R {
+
+	public static class layout {
+
+		public static final int activity_main = 0x7f06000a;
+	}
+}
```

Comment:
Because this is not an Android project, and the R files are not generated, those are mock files.